ConsentViewController/Classes/LocalStorage/SPUserDefaults.swift

```diff
@@ -17,6 +17,8 @@ class SPUserDefaults: SPLocalStorage {
     static let USER_DATA_KEY = "\(SP_KEY_PREFIX)userData"
```

Comment:
@Nevazhnovu in this class we don't need setters and getters, we can directly use properties (like the other attributes). Something like

```swift
    var gdprChildPMId: String? {
        get {
            storage.string(forKey: gdpr child PM id key)
        }
        set {
            storage.set(newValue, forKey: gdpr child PM id key)
        }
    }

    var ccpaChildPMId: String? {
        get {
            storage.string(forKey: gdpr child PM id key)
        }
        set {
            storage.set(newValue, forKey: gdpr child PM id key)
        }
    }
  ```
  
  And we can't forget to add those properties to the `SPLocalStorage` protocol.